template-compose/app/src/main/java/co/nimblehq/template/compose/di/modules/StorageModule.kt

```diff
@@ -17,18 +16,8 @@ abstract class StorageModule {
 
     companion object {
 
-        @Provides
-        fun provideDefaultSharedPreferences(@ApplicationContext context: Context): SharedPreferences =
-            PreferenceManager.getDefaultSharedPreferences(context)
-
         @Provides
         fun provideSecuredLocalStorage(@ApplicationContext context: Context) =
             EncryptedSharedPreferences(context)
```

Comment:
Hmmmm, sucks that there is no encrypted version of data-store 🥲

Reply:
@Tuubz Yeahhhh. I mentioned it in my blog too. 😁